docs/support/tutorials/gis/eo_guide.md

```diff
@@ -39,221 +54,166 @@ What to consider when chosing data:
+    
+[**NASA Earthdata**](https://search.earthdata.nasa.gov) provides among many others [harmonized Landsat 8 and Sentinel-2 dataset](https://hls.gsfc.nasa.gov/). It requires free registration and download is possible via web interface and bulk download.
 
+**[Amazon Web Service (AWS) open EO data](https://registry.opendata.aws/?search=tags:gis,earth%20observation,events,mapping,meteorological,environmental,transportation)** is a collection of worldwide EO datasets provided by different organizations, including Landsat and Sentinel. Some of the data can be downloaded only on "requestor pays" basis. The situation is changing all the time, currently [Sentinel-2 L2A Cloud-optimized Geotiffs](https://registry.opendata.aws/sentinel-2-l2a-cogs/) are available for free, also via STAC. 
 
-## How can I process the data?
+**[Microsoft planetary computer](https://planetarycomputer.microsoft.com)**: Sentinel, Landsat, MODIS etc. STAC available. It is currently available in preview.
 
-At CSC, EO data can be processed and analyzed using for example [CSC's supercomputer Puhti](../../../computing/systems-puhti.md) or a virtual machine in the [CSC cloud service cPouta](../../../cloud/pouta/pouta-what-is.md). You can find more information around geocomputing using CSC resources on [CSC geocomputing pages](https://research.csc.fi/geocomputing).
+[**Google Cloud Storage open EO data**](https://cloud.google.com/storage/docs/public-datasets), including Sentinel-2 L1C and Landsat: Collection 1 data. Data can be downloaded from here for example with [FORCE](../../../apps/force.md).
+    
+[**Terramonitor**](https://www.terramonitor.com/services/analysis-ready) provides pre-prosessed, analysis ready Sentinel-2 data from Finland available between 2018-2020. Commercial service.
 
-Puhti has a lot of applications already installed (see below), you do not need to worry about it. If you need further applications, you can ask us to install them for you. You can also add your own installations using for example the [CSC Tykky tool](../../../computing/containers/tykky.md).   In cPouta, you need to set up your own virtual machine including all security and software setup, see [CSC cPouta instructions](../../../cloud/pouta/launch-vm-from-web-gui.md).
+!!! default "Other geospatial datasets"
 
-!!! default "Please note"
+    To find other geospatial datasets, check out [CSC open spatial dataset list](https://research.csc.fi/open-gis-data).  
 
-    Puhti (and also CSC's other supercomputers Mahti and LUMI), work differently from your own computer. Before starting to use them, please familiarize yourself with the environment by for example visit the respective chapters in our [CSC computing environment self-learning course](https://ssl.eventilla.com/csccompenvselflearn).
 
-What to consider when chosing software:
+## How can I process EO data at CSC?
 
```

Comment:
```suggestion
* User skills and preferences
    * Graphical User Interface (GUI)
    * Command Line Interface (CLI)
    * Scripting
* User needs
    * Batch processing
    * Automation
    * Reproducibility
* Open source vs commercial
    * This guide focusses on [software available on CSC supercomputer Puhti](../../../apps/index.md) 
```

Reply:
I do not see that this list really helps. Scripting/CLI is mentioned elsewhere, for software people are still I think mostly looking for functionality and in bigger cases speed..

Reply:
I think it does, at it is not often thought about when choosing a tool. Especially the needs. And when growing up at university with commercial software one does not necessarily think about FOSS, until you end up working somewhere without all the nice licenses from univeristy. Luckily it is changing now slowly. 
Updated the text also.
What does it hurt to have it here?